charts/karmada/_crds/bases/config.karmada.io_resourceinterpretercustomizations.yaml

```diff
@@ -57,10 +57,12 @@ spec:
                           = \"v1\" dependency.kind = \"ServiceAccount\" dependency.name
                           = desiredObj.spec.serviceAccountName dependency.namespace
                           = desiredObj.namespace dependencies[0] = {} dependencies[0]
```

Comment:
int lua, the array index starts from 1, so I think it shall be `dependencies[1]=dependency`

Reply:
Fixed.